redbot/core/core_commands.py

```diff
@@ -4134,7 +4116,7 @@ async def command_disable_cog(self, ctx: commands.Context, *, cogname: str):
 
     @commands.guild_only()
     @command_manager.command(name="enablecog")
-    async def command_enable_cog(self, ctx: commands.Context, *, cogname: str):
+    async def command_enable_cog(self, ctx: commands.Context, *, cogname: CogConverter):
```

Comment:
As with command_name->command, this should also be renamed from cogname to cog:
```suggestion
    async def command_enable_cog(self, ctx: commands.Context, *, cog: CogConverter):
```
You will need to update the docstring and docs as well.